evm-contracts/src/v0.7/dev/LinkTokenReceiver.sol

```diff
@@ -3,10 +3,11 @@ pragma solidity ^0.7.0;
 
 abstract contract LinkTokenReceiver {
 
-  bytes4 constant private ORACLE_REQUEST_SELECTOR = 0x40429946;
   uint256 constant private SELECTOR_LENGTH = 4;
   uint256 constant private EXPECTED_REQUEST_WORDS = 2;
   uint256 constant private MINIMUM_REQUEST_LENGTH = SELECTOR_LENGTH + (32 * EXPECTED_REQUEST_WORDS);
+  bytes4 constant private ORACLE_REQUEST_SELECTOR = 0x40429946;
+  bytes4 constant private OPERATOR_REQUEST_SELECTOR = 0x6de879d6;
```

Comment:
This doesn't work since both `Oracle` and `Operator` implement `oracleRequest()` defined in `ChainlinkRequestInterface` (for backwards compatibility).

`requestOracleData()` is also not in the `OperatorInterface` because it is public, not external. It has to be public so that `oracleRequest()` can call from the same contract.